CUSTOM.md

```diff
@@ -0,0 +1,218 @@
+  - Once all elements are set press the "Update Elements" button
+- Presence track energy:
+  - For each time that energy gain increases grab energy tokens from bag and stack them onto spot
+  - You don't need to put them on every spot, only when income changes. Also don't forget about starting spot if it's not zero
+  - Energy tokens have a 2nd state to be used for bonus energy, i.e. Finder
+  - Once all energy is set press the "Update Energy" button
+- Innate thresholds:
+  - Grab a threshold token (X) and place it onto object where you want the threshold decal to appear
+  - Then grab elements from the element bag right next to threshold
+  - Set the state to match the element
+  - Stack elements on top of each other as needed
+  - Once all thresholds are set press the "Update Threshold" button (the top one)
+  - Note for thresholds with OR, i.e. Trickster, you'll need to manually edit the json file to add a second threshold at the exact same coordinates as the other one
+- Extra setup:
+  - Create object and place on top of spirit panel
+  - Tag object with "Setup"
+  - doSetup(params)
+    - params: table - contains data about player
+      - color: string - is the color of the player who picked the spirit
+
+Power Cards:
+- Set card onto power card editor
+- Change values that appear to match card
+- Set name of object to that of power card
+- Thresholds:
+  - Grab a threshold token (X) and place it onto object where you want the threshold decal to appear
+  - Then grab elements from the element bag right next to threshold
+  - Set the state to match the element
+  - Stack elements on top of each other as needed
+  - Once all thresholds are set press the "Update Threshold" button (the bottom one)
+  - Note that the decal will appear smaller than the actual size of the object, just a TTS limitation for how small an object can be
+  - Note for thresholds with OR, i.e. Trickster, you'll need to manually edit the json file to add a second threshold at the exact same coordinates as the other one
+
+Aspects:
+- Add "Aspect" tag
+- Set name of object to that of aspect
+- Thresholds:
+  - Place on power card editor
+  - Follow power card threshold instructions
+- Add "Aspect" tag to deck of aspects as well if you have one of those
+
+Blight Cards:
+- Set card onto blight card editor
+- Change values that appear to match card
+- Set name of object to that of blight card
+- If blight card has recurring start of invader effect set object's description to that
+- You'll also want to make sure that Hide Face Down toggle is set for the card
+
+Expansions:
+- Create a bag
+- Name the bag the name of the expansion
+- Add the tag "Expansion" to the bag
+- Decks of cards need to be named one of the following:
+  - "Fear"
+  - "Events"
+  - "Blight Cards"
+  - "Minor Powers"
+  - "Major Powers"
+- Note using single cards in place of a deck may cause setup issues
+
+Adversary & Scenario:
```

Comment:
Now that I think about it, perhaps each section should have a sentence describing the purpose of the section.
"Here you can setup pieces like Prussia 1 ability"
"Here you can adjust the invader deck like Prussia 2+"